shell/platform/windows/flutter_window.cc

```diff
@@ -277,4 +281,23 @@ PointerLocation FlutterWindow::GetPrimaryPointerLocation() {
   return {(size_t)point.x, (size_t)point.y};
 }
 
+void FlutterWindow::OnThemeChange() {
+  HIGHCONTRAST high_contrast = {sizeof(HIGHCONTRAST)};
+  if (SystemParametersInfoW(SPI_GETHIGHCONTRAST, sizeof(HIGHCONTRAST),
+                            &high_contrast, 0)) {
+    BOOL hc_on = high_contrast.dwFlags & HCF_HIGHCONTRASTON;
+    // Currently, only FlutterWindowsView should be used as delegate on Windows
+    FlutterWindowsView* view =
+        reinterpret_cast<FlutterWindowsView*>(binding_handler_delegate_);
+    FlutterWindowsEngine* engine = view->GetEngine();
+    engine->UpdateHighContrastEnabled(hc_on);
```

Comment:
Should we add a method to `WindowBindingHandlerDelegate` instead of accessing the engine directly here?

Reply:
Agreed - please update the binding handler interface for now. I promise I'll send out my cleanup and nuke that stuff later.